composer.json

```diff
@@ -20,15 +20,14 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "akeneo/pim-community-dev": "1.5.*"
+        "akeneo/pim-community-dev": "1.6.x-dev@dev"
```

Comment:
Just to keep in mind. Here I would put `~1.6` to avoid to tag a new version for each version of Akeneo PIM and be compatible with 1.7 if nothing has been changed in this bundle scope.